### PR TITLE
 update user controller and router

### PR DESCRIPTION
**Pull request by hewonjeong**
- update user controller and router

### CODE REVIEW

**Comment by ovekyc** [comment]
LGTM

File: src/server/users/controllers.js [hunk 1]
```diff
@@ -69,5 +71,34 @@ export default {
     }).catch((err) => {
       return cb(new APIError(err, {statusCode: err.statusCode, message: err.message}));
     });
+  },
+  addCreatedPost(req, res, cb) {
+    const currentTime = new Date();
```
Comment (festinalente91) [inline src/server/users/controllers.js]: This is unneccessary. 
KeyUtils.genTimeHash() use current time by default. :)

File: src/server/users/controllers.js [hunk 2]
```diff
@@ -69,5 +71,34 @@ export default {
...
+      }
+      res.status(200).send({
+        message: 'success',
+        data: idxKey
+      });
+      return cb();
+    });  
+  },
+  addSavedPost(req, res, cb) {
+    const currentTime = new Date();
```
Comment (festinalente91) [inline src/server/users/controllers.js]: ditto

File: src/server/users/controllers.js (hunk 1 above)
Comment (HewonJeong) [inline src/server/users/controllers.js]: KeyUtils.genTimeHash() need a parameter, Time object.
cause in item controller, we make Time object to make createdDate field.
timehash's time should be same with this createdDate's time so we couldn't make Time object in KeyUtils function.

File: src/server/users/controllers.js (hunk 2 above)
Comment (HewonJeong) [inline src/server/users/controllers.js]: ditto lol

**Comment by festinalente91** [comment]
LGTM

**Pull request merged**